docs/features/pointing_device.md

```diff
@@ -927,3 +928,827 @@ In general the following two functions must be implemented in appropriate locati
+| `pointing_modes_set_residuals(residuals)`                              | Set active device's residuals(_accumulated x, y_) to `residuals`                                                           | _None_                       | `pointing_modes_residuals_t` |
+| `pointing_modes_get_residuals()`                                       | Get active device's residuals                                                                                              | `pointing_modes_residuals_t` | _None_                       |
+| `pointing_modes_reset()`                                               | reset active device to default values and clear residuals                                                                  | _None_                       | _None_                       |
+| `pointing_modes_get_divisor()`                                         | Get current precision adjusted divisor                                                                                     | `mouse_xy_report_t`          | _None_                       |
+| `pointing_modes_get_direction()`                                       | Get current direction                                                                                                      | `uint8_t`                    | _None_                       |
+| `pointing_modes_get_type()`                                            | Get current mode type and options                                                                                          | `uint8_t`                    | _None_                       |
+| `pointing_modes_update()`                                              | Recalculate direction, precision adjusted final divisor, and mode_type                                                     | _None_                       | _None_                       |
+| `pointing_modes_apply_divisor(axis)`                                   | Returns result of applying `divisor` to `residuals` based on `axis` (_[see advanced modes](#Pointing-Modes-Advanced-use)_) | `mouse_xy_report_t`          | `uint8_t`                    |
+| `pointing_modes_get_active_device()`                                   | Returns device id of current active device (_see controlling multiple devices_)                                            | `uint8_t`                    | _None_                       |
+| `pointing_modes_set_active_device(device_id)`                          | Set active device to `device_id` (_see controlling multiple devices_)                                                      | _None_                       | `uint8_t`                    |
+
+***Note: `pointing_modes_get_divisor`, `pointing_modes_get_direction`, `pointing_modes_get_mode_type` will not recalculate any of the values to reflect changes in `mode_id` or `residuals` made in the same block of code, `pointing_modes_update` must be used first***
```

Comment:
```suggestion
::: tip
`pointing_modes_get_divisor`, `pointing_modes_get_direction`, `pointing_modes_get_mode_type` will not recalculate any of the values to reflect changes in `mode_id` or `residuals` made in the same block of code, `pointing_modes_update` must be used first
:::
```

Maybe a `tip` block fits better? If so please update the other usages of `***Note:`